code-monkey/src/util/logs.py

```diff
@@ -6,11 +6,13 @@ def get_logger(name):
     logger = logging.getLogger(f"codemonkey:{name}")
     return logger
 
+MAX_LENGTH=200
+
 class TruncatingFormatter(colorlog.ColoredFormatter):
     def format(self, record):
         message = super().format(record)
-        if len(message) > 100:
-            return message[:97] + "...\033[0m"
+        if len(message) > MAX_LENGTH:
```

Comment:
I later realized that we only want to truncate if its a DEBUG msg, maybe if INFO, but not above that level. I can fix that later though